libs/remix-ui/terminal/src/lib/components/RenderUnknownTransactions.tsx

```diff
@@ -5,6 +5,7 @@ import Context from './Context' // eslint-disable-line
 import showTable from './Table'
 
 const RenderUnKnownTransactions = ({ tx, receipt, index, plugin, showTableHash, txDetails, modal, provider }) => {
```

Comment:
don't you need to do the same changes in the `RenderKnownTransaction.tsx`?

Reply:
updated